packages/teleport-project-generator-html/src/index.ts

```diff
@@ -20,9 +20,12 @@ const createHTMLProjectGenerator = (config?: { individualEntyFile: boolean }) =>
     pages: {
       generator: createHTMLComponentGenerator,
       path: [''],
+      options: {
+        useFileNameForNavigation: true,
+      },
     },
     static: {
-      prefix: 'public',
+      prefix: '/public',
```

Comment:
Can you check this, we don't need to add `/` the server takes care of it i think